source/extended-json.rst

```diff
@@ -5,13 +5,13 @@ Extended JSON
 :Spec: 587
 :Spec-ticket: SPEC-587
 :Title: Extended JSON
-:Author: Luke Lovett
+:Author: David Golden
```

Comment:
I concur, although it should be noted that @TylerBrock has apparently been retconned out of existence on the [CRUD spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst).